addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1161,9 +1161,12 @@ const Wysiwyg = Widget.extend({
                 }
                 this.linkTools.noFocusUrl = options.noFocusUrl;
                 const _onClick = ev => {
+                    const selection = this.odooEditor.document.getSelection();
+                    const isFocusOnInput = closestElement(selection.anchorNode, '.o_url_input');
```

Comment:
No, In case of `mass_mailing` it will not.

Reply:
@adsa-odoo For 16.0 and 17.0 you are right, because from saas-17.2 onwards @deso-odoo will be right. Does the issue also occur in saas-17.2+ ? If yes, you'll need to find another way to fix it.

Reply:
Well, yes, you are right that you'll need to find another fix for saas-17.2+, but since I don't like the fact that we're passing an event to `destroyLinkTools` (see below), I wonder if maybe it wouldn't be better to try to find a fix for saas-17.2 first, hoping it can be a better fix, then see if we can backport it to 16.0 in some way?